editor/src/clj/editor/keymap.clj

```diff
@@ -30,127 +30,129 @@
   ;; of mistake.
   ;; Note that specifying Shortcut key (which is Meta on mac and Ctrl on windows
   ;; and linux) is not allowed.
-  {:macos   [["A" :add]
```

Comment:
Fixed indentation for the `:macos` block (due to earlier rename from `:darwin`), but also added `["Alt+Meta+Y" :async-reload]`. This key combination used to perform the same operation in IntelliJ (not sure they have manual refresh anymore?) and is suitably unique for our needs.